Dockerfile_dev

```diff
@@ -0,0 +1,61 @@
+FROM jupyter/base-notebook:eb70bcf1a292
```

Comment:
Ditto

Reply:
Done but as great professional :sweat_smile: I tested this dockerfile locally and although it builds everything perfectly when I run it I got the error I told you some meetings ago: `ReferenceError: React is not defined` which locally I solved by adding 
```
 new webpack.ProvidePlugin({
            "React": "react",
        }),
```
to the webpack.config.js, but for the given scenario I don't know exactly how to proceed.